nix/ormolize/default.nix

```diff
@@ -1,15 +1,20 @@
-{ pkgs, haskellPackages }: _: p:
+{ pkgs, haskellPackages }: { package, doCheck ? false }:
   pkgs.stdenv.mkDerivation {
-    name = p.name + "-ormolized";
-    src = p.src;
+    name = package.name + "-ormolized";
+    src = package.src;
     buildInputs = [
       haskellPackages.cpphs
-      (pkgs.haskell.lib.dontCheck haskellPackages.ormolu)
+      haskellPackages.ormolu
```

Comment:
If we leave `dontCheck` here then CI will build ormolu twice; once with tests enabled, once with disabled.

Reply:
I think this is OK. The reason we have `donetCheck` here is that we can try how things work without running tests on every rebuild, this is very annoying.